github/CheckSuite.py

```diff
@@ -0,0 +1,241 @@
+        return self._app.value
+
+    @property
+    def before(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._before)
+        return self._before.value
+
+    @property
+    def check_runs_url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._check_runs_url)
+        return self._check_runs_url.value
+
+    @property
+    def conclusion(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._conclusion)
+        return self._conclusion.value
+
+    @property
+    def created_at(self):
+        """
+        :type: datetime.datetime
+        """
+        self._completeIfNotSet(self._created_at)
+        return self._created_at.value
+
+    @property
+    def head_branch(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._head_branch)
+        return self._head_branch.value
+
+    @property
+    def head_commit(self):
+        """
+        :type: :class:`github.GitCommit.GitCommit`
+        """
+        self._completeIfNotSet(self._head_commit)
+        return self._head_commit.value
+
+    @property
+    def head_sha(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._head_sha)
+        return self._head_sha.value
+
+    @property
+    def id(self):
+        """
+        :type: int
+        """
+        self._completeIfNotSet(self._id)
+        return self._id.value
+
+    @property
+    def latest_check_runs_count(self):
+        """
+        :type: int
+        """
+        self._completeIfNotSet(self._latest_check_runs_count)
+        return self._latest_check_runs_count.value
+
+    @property
+    def pull_requests(self):
+        """
+        :type: list of :class:`github.PullRequest.PullRequest`
+        """
+        self._completeIfNotSet(self._pull_requests)
+        return self._pull_requests.value
+
+    @property
+    def repository(self):
+        """
+        :type: :class:`github.Repository.Repository`
+        """
+        self._completeIfNotSet(self._repository)
+        return self._repository.value
+
+    @property
+    def status(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._status)
+        return self._status.value
+
+    @property
+    def updated_at(self):
+        """
+        :type: datetime.datetime
+        """
+        self._completeIfNotSet(self._updated_at)
+        return self._updated_at.value
+
+    @property
+    def url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._url)
+        return self._url.value
+
+    def rerequest(self):
+        """
+        :calls: `POST /repos/:owner/:repo/check-suites/:check_suite_id/rerequest <https://docs.github.com/en/rest/reference/checks#rerequest-a-check-suite>`_
+        :rtype: bool
+        """
+        request_headers = {"Accept": "application/vnd.github.v3+json"}
```

Comment:
Is this from removing the preview headers? Since isn't this the default?

Reply:
Yes, these are the default headers after removing the preview headers. If you want I can remove them, I will also have to remove them from the replay data files.

Reply:
I see, they are not in the default headers. Objection removed.